.travis.yml

```diff
@@ -47,7 +51,14 @@ install:
 
 script:
   - cargo generate-lockfile
-  - ci/run-docker.sh $TARGET $FEATURES
+  - |
+    if [ "$NORUN" == "1" ]; then
+      cd crates/stdsimd
```

Comment:
Could this use the `-p` argument to avoid changing the cwd?

Reply:
Sure thing.